### PR TITLE
E2-1883: Remove displayed event 

### PR DESCRIPTION
**Pull request by e7mac**
What              | Where/Who
------------------|----------------------------------------
JIRA Issue        | [E2-1883](https://leanplum.atlassian.net/browse/E2-1883)
People Involved   | @shivanshukumar 

## Background
Need to remove an event leads to inconsistent results on the dashboard

## Implementation
Remove event

## Testing steps
None

## Is this change backwards-compatible?
Yes

### CODE REVIEW

**Comment by e7mac** [comment]
Added the links to the JIRA. There's no test necessary since we're just removing an event.

File: Leanplum-SDK/Classes/Features/Actions/LPActionManager.m [hunk 1]
```diff
@@ -518,18 +518,6 @@ - (BOOL)hasTrackedDisplayed:(NSDictionary *)userInfo
     return NO;
 }
 
-- (void)trackMessageDisplayed:(NSDictionary *)userInfo
```
Comment (shivanshukumar) [inline Leanplum-SDK/Classes/Features/Actions/LPActionManager.m]: @e7mac you sure this is an internal method?

Comment (hrishileanplum) [reply Leanplum-SDK/Classes/Features/Actions/LPActionManager.m]: I approved it and then noticed Shiv has a question :).

Comment (hrishileanplum) [reply Leanplum-SDK/Classes/Features/Actions/LPActionManager.m]: I approved it and then noticed Shiv has a question :).

**Pull request merged**